dev/io.openliberty.jakarta.pages.3.0/bnd.bnd

```diff
@@ -14,13 +14,15 @@ bVersion=1.0
 Bundle-SymbolicName: io.openliberty.jakarta.pages.3.0; singleton:=true
 Bundle-Description: Jakarta Server Pages, version 3.0
 
+# Note: No files under jakarta.servlet.jsp.resources, but it's needed to resolve other jsp bundles (i.e. com.ibm.jsp.2.3.jakarta)
 Export-Package: \
 	jakarta.servlet.jsp; version="3.0.0",\
 	jakarta.servlet.jsp.el; version="3.0.0",\
-	jakarta.servlet.jsp.resources; version="3.0.0",\
-	jakarta.servlet.jsp.tagext; version="3.0.0"
+	jakarta.servlet.jsp.tagext; version="3.0.0",\
+	jakarta.servlet.jsp.resources; version="3.0.0"
```

Comment:
any reason the order was changed here?

Reply:
Not intentional. I had added a comment between the exports but then opted to place it above on line 17 instead.

Reply:
Can we change it back? It makes looking for a package easier If they are in alphabetical order.

Reply:
Changed to order order